descope/management/sso_settings.py

```diff
@@ -0,0 +1,156 @@
+from typing import List
+
+from descope.auth import Auth
+from descope.management.common import MgmtV1
+
+
+class RoleMapping:
+    def __init__(self, groups: List[str], role_name: str):
+        self.groups = groups
+        self.role_name = role_name
+
+
+class SSOSettings:
+    _auth: Auth
+
+    def __init__(self, auth: Auth):
+        self._auth = auth
+
+    def configure(
+        self,
+        mgmt_key: str,
+        tenant_id: str,
+        enabled: bool,
+        idp_url: str,
+        entity_id: str,
+        idp_cert: str,
+        redirect_url: str,
```

Comment:
I also dont see a way to configure the "Tenant domain" (I must admit that I dont remember what it use for.. (its not the domain you set when creating the tenant))
<img width="585" alt="image" src="https://user-images.githubusercontent.com/105154237/203432710-78665c9e-c72f-427e-bc14-3b81c1501b58.png">

Reply:
Yeah. This might not be used at all at the moment. Left out for now after consulting.